src/platform/pantherlake/include/platform/lib/pm_runtime.h

```diff
@@ -0,0 +1,72 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2022 Intel Corporation. All rights reserved.
+ *
+ * Author: Tomasz Lauda <tomasz.lauda@linux.intel.com>
+ */
+
+/**
+ * \file platform/tigerlake/include/platform/lib/pm_runtime.h
```

Comment:
Funny how a wrong doxygen unintentionally provides useful information: it points at the origin of the copy/paste/diverge.